src/views/Manager/index.js

```diff
@@ -82,27 +81,27 @@ const Manager = () => {
     },
   ];
 
-const updateManager = (payload) => {
-  axios
-    .patch(`${process.env.REACT_APP_PROXY}/api/user/${id}`, 
-    payload, 
-    { Authorization: `Bearer ${userContext.user.accessJwt}` }
-    )
-    .then(response => {
-      setManager({
-        ...managerData,
-        firstName: response.data.firstName,
-        lastName: response.data.lastName,
-        phone: response.data.phone,
-        email: response.data.email,
-      });
-      setEditingStatus(false);
-      Toast("Save successful!");
-    })
-    .catch((error) => {
+  const updateManager = (payload) => {
+    axios
+      .patch(`${process.env.REACT_APP_PROXY}/api/user/${id}`,
+        payload,
+        { Authorization: `Bearer ${userContext.user.accessJwt}` }
+      )
```

Comment:
Authorization object needs to be associated with a header key:
`{ headers: { 'Authorization': `Bearer ${context.user.accessJwt}` } }`
For a DRY way to do this, check out some of the files that use the `makeAuthHeaders` helper function